pkg/ccl/backupccl/backup_destination.go

```diff
@@ -309,7 +309,7 @@ func readLatestFile(
 		return "", err
 	}
 	defer collection.Close()
-	latestFile, err := collection.ReadFile(ctx, latestFileName)
+	latestFile, err := collection.ReadFile(ctx, latestHistoryDirectory+"/"+latestFileName)
```

Comment:
In the future "latestFileName" will have a version on it which is why I have them as two separate consts but if it's clunky I can easily make it one const just for this PR.